src/bin/wasm-tools/component.rs

```diff
@@ -116,7 +116,7 @@ pub struct NewOpts {
     ///
     /// If the old import name is not found, it is ignored.
     #[clap(long = "import-name", value_name = "[OLD]=NEW", value_parser = parse_import_name)]
-    import_names: HashMap<String, String>,
+    import_names: Option<HashMap<String, String>>,
```

Comment:
I don't think this will work as intended; originally I expected clap to treat `HashMap` as `Vec` with respect to multivalue options.

I think we'll need to have `Vec<(String, String)>` and do a `self.import_names.into_iter().collect()` on it.

Reply:
```suggestion
    import_names: Vec<(String, String)>,
```